src/Scrapers/Pipeline/PipelineRegistry.ts

```
@@ -0,0 +1,24 @@
+ * Banks register here as they are migrated.
+ */
+
+import type { CompanyTypes } from '../../Definitions.js';
+import { CompanyTypes as CT } from '../../Definitions.js';
+import type { ScraperOptions } from '../Base/Interface.js';
+import { buildDiscountPipeline } from '../Discount/DiscountPipeline.js';
+import { buildVisaCalPipeline } from '../VisaCal/VisaCalPipeline.js';
+import type { IPipelineDescriptor } from './PipelineDescriptor.js';
+
+/** Factory that builds a pipeline descriptor for a specific bank. */
+type PipelineFactory = (options: ScraperOptions) => IPipelineDescriptor;
```

ESLint

[error] 'options' is defined but never used.

(no-unused-vars)